.github/workflows/release.yml

```diff
@@ -38,4 +38,3 @@ jobs:
     - name: Publish static assets to S3
       run: |
         aws s3 cp build/ s3://getunleash-static/unleash/${{ steps.get_version.outputs.VERSION }} --recursive
```

Comment:
`.editorconfig` has `insert_final_newline = true`. weird that it is removed here

Reply:
Looks like there were two trailing newlines in main.